util/reggen/validate.py

```diff
@@ -1275,16 +1275,19 @@ def validate_multi(mreg, offset, addrsep, width, top):
     return error, rnum
 
 
-def make_intr_reg(regs, name, offset, swaccess, hwaccess, desc):
-    intrs = regs['interrupt_list']
+def make_intr_alert_reg(regs, name, offset, swaccess, hwaccess, desc):
+    if name == 'ALERT_TEST':
+        signal_list = regs['alert_list']
+    else:
+        signal_list = regs['interrupt_list']
     genreg = {}
     genreg['name'] = name
     genreg['desc'] = desc
-    genreg['hwext'] = 'true' if name == 'INTR_TEST' else 'false'
-    genreg['hwqe'] = 'true' if name == 'INTR_TEST' else 'false'
+    genreg['hwext'] = 'true' if name in ['INTR_TEST', 'ALERT_TEST'] else 'false'
+    genreg['hwqe'] = 'true' if name in ['INTR_TEST', 'ALERT_TEST'] else 'false'
     genreg['hwre'] = 'false'
     # Add tags.
-    if name == 'INTR_TEST':
+    if name in ['INTR_TEST', 'ALERT_TEST']:
```

Comment:
Maybe it would be nice to define a variable for `name in ['INTR_TEST', 'ALERT_TEST']` and use it here? That way, the next magic test only needs a 1-line change.

Reply:
yes, let me do that.